lib/build-image.sh

```diff
@@ -84,7 +84,7 @@ EOF
         shadow_unit="$(base64 -w0 etc-shadow.mount)"
         cat <<EOF >"${TMPDIR}/pc/cloud.conf"
 #cloud-config
-datasource_list: [NoCloud]
+datasource_list: [NoCloud,None]
```

Comment:
Should be enough with `[None]`, at least that worked in my last tests and we have already changed snapd tests to this.

Reply:
OK, I will test it.

Reply:
That worked. Thank you.